assets/panel/panel.js

```diff
@@ -50,6 +50,11 @@ DebuggerPanel.prototype = {
             .setNodeFront(front, { reason: "debugger" });
 
           return Promise.all([onNodeFrontSet, onInspectorUpdated]);
+        }.bind(this),
+        openConsoleAndEvaluate: async function(input) {
+          await this.toolbox.selectTool("webconsole");
+          const jsterm = this.toolbox.getPanel("webconsole").hud.jsterm;
```

Comment:
this could be a bit slow, but because this is an infrequent action, it might be better than spinning up our own jsterm

Reply:
Agreed.